pkg/apis/batch/validation/validation_test.go

```diff
@@ -654,6 +654,19 @@ func TestValidateJob(t *testing.T) {
 				Template:    validPodTemplateSpecForGenerated,
 			},
 		},
+		"spec.backoffLimit:must be greater than or equal to 0": {
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      "myjob",
+				Namespace: metav1.NamespaceDefault,
+				UID:       types.UID("1a2b3c"),
+			},
+			Spec: batch.JobSpec{
+				BackoffLimit: &negative,
```

Comment:
it looks like this variable already existed, but following the DAMP principle, this should be `pointer.Int64(-1)`

Reply:
DAMP?

Reply:
TIL: Descriptive And Meaningful Phrases

Reply:
https://stackoverflow.com/questions/6453235/what-does-damp-not-dry-mean-when-talking-about-unit-tests